pkg/services/alerting/extractor.go

```diff
@@ -12,6 +13,12 @@ import (
 	"github.com/grafana/grafana/pkg/models"
 )
 
+var (
+	valueFormatRegex = regexp.MustCompile(`^\d+`)
+	unitFormatRegex  = regexp.MustCompile(`[a-z]+`)
```

Comment:
Any reason changing this from `unitFormatRegex = regexp.MustCompile(`\w{1}$`)`?

Reply:
Yeah, so check this case out:

* _using `\w{1}$`_, when checking with `5hh`, the captured letter is `h`, which can (falsely) be valid, since it takes only the last character.
* _using `[a-z]+`_, when checking with `5hh`, the captured characters are `hh` which is invalid and should throw an error.